src/main/java/tk/shanebee/hg/listeners/GameListener.java

```diff
@@ -447,6 +456,27 @@ private void onInteract(PlayerInteractEvent event) {
 		}
 	}
 
+	@EventHandler
+	private void onInventoryClick(InventoryClickEvent e){
+		if (e.getWhoClicked() instanceof Player){
```

Comment:
Watch your spacing here
```suggestion
	private void onInventoryClick(InventoryClickEvent e) {
		if (e.getWhoClicked() instanceof Player) {
```